src/calculation/Calculator.ts

```diff
@@ -179,6 +180,21 @@ export async function calculate<T extends CalculationOptions>(
     debugObject.detailedResults = executionResults;
   }
 
+  if (options.calculateClauseCoverage) {
+    const clauseHTML = generateClauseCoverageHTML(elmLibraries, executionResults);
+    if (debugObject && options.enableDebugOutput) {
+      const debugHtml = {
+        name: 'clause-coverage.html',
```

Comment:
The debug output for logic highlighting pertains to an individual patient rather than all the patients being fed in, whereas the clause coverage HTML applies highlighting based on the results of all the inputted patients. From my understanding of the code, this means that I have access to the `detailedResult.groupId` for each of the patients' detailed results, but not one overall "group Id" that I can use here. I could pull off the `groupId` for each of the patients that are included in the `executionResults` array that gets fed into the `generateClauseCoverageHTML()` function. Do you think that would be helpful?

Reply:
I'm fine with just keeping it `clause-coverage.html`

Reply:
Gotcha gotcha that makes sense yeah don't change that thanks for clarifying!!